test/unit/dns.server.test.ts

```diff
@@ -4,125 +4,82 @@ import {
   upsertRecord,
   deleteRecord,
   getChangeStatus,
-  RecordType,
 } from '~/lib/dns.server';
+import { RecordType } from '@prisma/client';
 
 describe('DNS server lib function test', () => {
   let hostedZoneId: string;
+  const rootDomain = 'starchart.com';
 
   beforeAll(async () => {
-    hostedZoneId = await createHostedZone('starchart.com');
+    hostedZoneId = await createHostedZone(rootDomain);
+    process.env.AWS_ROUTE53_HOSTED_ZONE_ID = hostedZoneId;
   });
 
-  afterAll(() => resetHostedZone());
-
   test('Hosted zone is created and hosted zone is returned', () => {
     expect(hostedZoneId.length).toBeGreaterThan(0);
     expect(/^[A-Z0-9]+/.test(hostedZoneId)).toBe(true);
   });
 
   test('createRecord() creates a record in existing hosted zone and returns changeId', async () => {
-    const changeId = await createRecord(
-      hostedZoneId,
-      RecordType.A,
-      'my-domain.starchart.com',
-      '192.168.0.1'
-    );
+    const changeId = await createRecord(RecordType.A, `my-domain.${rootDomain}`, '192.168.0.1');
     expect(changeId?.length).toBeGreaterThan(1);
   });
 
-  test('createRecord() throws an error when invalid hosted zone ID or name or value ', async () => {
-    await expect(
-      createRecord('WRONGHOSTEDZONEID', RecordType.A, 'sub2.starchart.com', '192.168.0.2')
-    ).rejects.toThrow();
+  test('createRecord() throws an error when name or value is provided ', async () => {
```

Comment:
I don't understand this.  Why does it through if the name or value is provided?  Can we clarify what this test is actually testing?

Reply:
It was `invalid hostedZoneId or name...` I removed `invalid` by mistake. Sorry fro confusion.